bika/lims/utils/__init__.py

```diff
@@ -143,14 +143,8 @@ def getUsers(context, roles, allow_empty=True):
 def isActive(obj):
     """ Check if obj is inactive or cancelled.
     """
-    wf = getToolByName(obj, 'portal_workflow')
-    if (hasattr(obj, 'inactive_state') and obj.inactive_state == 'inactive') or \
-       wf.getInfoFor(obj, 'inactive_state', 'active') == 'inactive':
-        return False
-    if (hasattr(obj, 'cancellation_state') and obj.inactive_state == 'cancelled') or \
-       wf.getInfoFor(obj, 'cancellation_state', 'active') == 'cancelled':
-        return False
-    return True
+    from bika.lims.workflow import isActive
+    return isActive(obj)
```

Comment:
@xispa The `@deprecated` decorator cannot be used here because, if imported, it results in a circular import.  `utils/__init__.py` imports `lims/__init.py` which imports `validators.py` that imports again `utils/__init__.py`